app/models/stats/claim_stats.rb

```diff
@@ -0,0 +1,11 @@
+module Stats
+  class ClaimStats < Claim
+    scope :started, -> { where.not(password_digest: nil, state: 'submitted').within_timeframe }
+    scope :started_count, -> { started.count }
+
+    scope :completed, -> { where(state: 'submitted').within_timeframe }
+    scope :completed_count, -> { completed.count }
+
+    scope :within_timeframe, -> { where('created_at > ?', 92.days.ago) }
```

Comment:
@TrentDSD my subtext was that it should be a constant.

92 is an interesting number, though: it's not a multiple of 3; it's the maximum of all three-month sequences (30 + 31 + 31). I'd put that in the source.